[js] add documentation on dest properties (#825)
SAP/cloud-sdk: docs/js/features/connectivity/destination.mdx

  The value is in milliseconds and the default value is 10000 (10 seconds).
  There is a second timeout options on the request level, setting the timeout for the calls to the destination target.

## Destination Properties

The destination object may contain additional properties.
The properties change the behavior how the SAP Cloud SDK handles the HTTP request at runtime.

### SAP Client

> The property `sap-client` is considered by the SAP Cloud SDK.
> This property leads to a header field `sap-client` with the specified value in the HTTP request.

Reviewer: [pp] "This property leads to" sounds a bit strange - The SDK does something with the property, which is just there.

```suggestion
The property `sap-client` is considered by the SAP Cloud SDK.
When this property is set, it is used as the header parameter `sap-client` with the specified value in the HTTP request to the target system.
```

<img
  alt="SAP client property on destination"
  src={useBaseUrl('img/destination-property-sap-client.png')}
  class="center"

> />

Reviewer: [q] General: I am not sure if it is worth creating images for all of these. Do you think it would be enough to just have one for all of them in the introduction of this section?

Author: I think so too. I will take one image and remove the other

Author: ALso removed the other trust pic because it was to much.

Reviewer: 👍

### Trust Configuration

> There are two destination property used to establish trust to systems using certificates not issued by trusted certificate authorities:

Reviewer: [req] Typo
[req] The sentence is hard to read

```suggestion
By default, SAP BTP only trusts certain certificate authorities.
If you want to make HTTPS requests against systems that use certificates from other certificate authorities, you can configure the following properties:
```

1. TrustStoreLocation: The SAP Cloud SDK adds the provided certificate in the `ca` property of the [node client](https://nodejs.org/api/tls.html#tlscreatesecurecontextoptions).
2. TrustAll: The SAP Cloud SDK adds the inverted value as the [rejectUnauthorized](https://nodejs.org/api/tls.html#class-tlstlssocket)

For additional information on trust configuration have a look at the more [detailed guide](../../guides/trust-and-key-stores.mdx).

<img
  alt="Truststore location property on destination"
  src={useBaseUrl('img/destination-property-truststore.png')}
  class="center"
/>

> <br />

Reviewer: [q] I think to keep the theme consistent everywhere, we shouldn't do this. How bad is it?
```suggestion
```

Author: Then there is no line break between the pictures.
